hack/otel/usage.md

```diff
@@ -0,0 +1,28 @@
+# Sample stack for testing OTEL functionality with moby
+
+To easily test the OTEL functionality present in moby, you can spin up a small demo compose stack that includes:
+- an OTEL collector container;
+- a Jaeger container to visualize traces;
+- an alternative Aspire Dashboard container to visualize traces;
+
+The OTEL collector is configured to export Traces to both the Jaeger and Aspire containers.
+
+The `hack/otel` directory contains the compose file with the services configured, along with a basic configuration file for the OTEL collector.
+
+## How can I use it?
+
+1) Export the env var used to override the OTLP endpoint:  
```

Comment:
Nit; should this be a actual markdown numbered list?

Reply:
Mmh good question, I thought it looked a bit better but I guess it depends on the markdown renderer used. No hard feelings here, if it's "more correct" to do otherwise i'll change this

Reply:
sounds good to me! I'll fix up this pr today since it's been lingering for a while

Reply:
I very much forgot it was still there 😂 ❤️